app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -0,0 +1,83 @@
+module Katello
+  class Api::V2::HostSubscriptionsController < Katello::Api::V2::ApiController
+    before_filter :find_host
+    before_filter :check_subscriptions, :only => [:add_subscriptions, :remove_subscriptions]
+
+    api :GET, "/hosts/:host_id/subscriptions", N_("List a host's subscriptions")
+    param :host_id, Integer, :desc => N_("Id of the host"), :required => true
+    def index
```

Comment:
I like your suggestion better than what you have now. However, I still worry about it being inconsistent with the rest of the api. Typically, if I can do `/content_views/5/repositories`, I can also do `/repositories?content_view_id=5`. This makes also easier to add/remove additional filters like `/repositories?content_view_id=5&content_type=docker` (and then if I want just the content_type filter, all I have to do is drop content_view_id). I guess my question would then be why don't we just add quantity in the subscriptions controller instead of defining a new index action here?